src/template.yaml

```diff
@@ -448,6 +448,66 @@ Actor:
         value: ''
       vulnerability:
         value: ''
+    resources:
+      perCombat: {}
+      spendable:
+        stoke:
+          enabled: false
+          current: 0
+        custom1:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom2:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom3:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom4:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom5:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom6:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom7:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom8:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
+        custom9:
+          label: ""
+          current: 0
+          max: 0
+          enabled: false
+          rest: "none"
```

Comment:
These are duplicates of the character actor specific data - shouldn't both be moved to the standard templated (and therefore defined only once)?

Reply:
Yep, I had reimplemented since the non-custom resources were different, but I had forgotten that it merges in the base templates. Will get this corrected 👍